Packs/RecordedFuture/Integrations/RecordedFuture/RecordedFuture.py

```diff
@@ -532,33 +529,54 @@ def build_intel_markdown(entity_data: Dict[str, Any], entity_type: str) -> str:
 
 
 def build_intel_context(entity: str, entity_data: Dict[str, Any],
-                        entity_type: str) \
-        -> Tuple[List[Common.Indicator], Dict[str, Any]]:
+                        entity_type: str, markdown: str) \
+        -> List[CommandResults]:
     """Build Intelligence context."""
     if entity_type == 'hash':
         entity_type = 'file'
     elif entity_type == 'vulnerability':
         entity_type = 'cve'
     indicators = []
+    command_results: List[CommandResults] = []
     if entity_data and ('error' not in entity_data):
         data = entity_data['data']
-        indicators.append(
-            create_indicator(entity, entity_type, data['risk']['score'],
-                             data['entity'].get('description', ''),
-                             location=data.get('location', None)))
-        indicators.extend([
-            create_indicator(x['ip']['name'], 'ip', x['score'],
-                             f'IP in the same CIDR as {entity}')
-            for x in data.get('riskyCIDRIPs', [])])
         data.update(data.pop('entity'))
         data.update(data.pop('risk'))
         data.update(data.pop('timestamps'))
         if data.get('relatedEntities', None):
             data['relatedEntities'] = handle_related_entities(
                 data.pop('relatedEntities'))
+
+        indicator = create_indicator(entity, entity_type, data['risk']['score'],
+                                     data['entity'].get('description', ''),
+                                     location=data.get('location', None))
+        command_results.append(CommandResults(outputs_prefix=get_output_prefix(entity_type),
+                                              outputs=data,
+                                              raw_response=entity_data,
+                                              readable_output=markdown,
+                                              outputs_key_field='name',
+                                              indicator=indicator))
+
+        for x in data.get('riskyCIDRIPs', []):
+            indicator = create_indicator(x['ip']['name'], 'ip', x['score'],
+                                         f'IP in the same CIDR as {entity}')
+            command_results.append(CommandResults(outputs_prefix=get_output_prefix(entity_type),
+                                                  outputs=data,
+                                                  raw_response=entity_data,
+                                                  readable_output=markdown,
+                                                  outputs_key_field='name',
+                                                  indicator=indicator))
```

Comment:
Please omit most of these keys - `outputs` with data is created above, so you should just use the `indicator` key, and add a new `readable_output`.